sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/implementation/converters/RoleTypeConverter.java

```diff
@@ -3,62 +3,62 @@
 
 package com.azure.communication.rooms.implementation.converters;
 
-import com.azure.communication.rooms.models.RoleType;
+import com.azure.communication.rooms.models.ParticipantRole;
```

Comment:
This file can be deleted since there is no more RoleType, we've directly renamed that to ParticipantRole